comp/core/flare/component.go

```diff
@@ -32,7 +33,10 @@ type Component interface {
 
 // Module defines the fx options for this component.
 func Module(params Params) fxutil.Module {
+	var fbf types.FlareBuilderFactory = helpers.NewFlareBuilder
+
 	return fxutil.Component(
 		fx.Provide(newFlare),
-		fx.Supply(params))
+		fx.Supply(params),
+		fx.Supply(fbf))
```

Comment:
Maybe we can include the flare profile component here, since it looks like all the command entrypoint that we include the flare component we also include the flare profiler. What do you think?